Sources/Basic/FSProxy.swift

```diff
@@ -268,19 +268,19 @@ private class LocalFS: FSProxy {
 //
 // FIXME: This class does not yet support concurrent mutation safely.
 public class PseudoFS: FSProxy {
-    private class Node {
+    fileprivate class Node {
```

Comment:
It does seem wrong to use `file` to mean `scope`, but it was the name the community settled on 😅